tests/onnxruntime/test_modeling.py

```diff
@@ -1871,7 +1875,10 @@ def test_compare_to_io_binding(self, model_arch):
         self.assertIsInstance(io_outputs.logits, torch.Tensor)
 
         # compare tensor outputs
-        self.assertTrue(torch.equal(onnx_outputs.logits, io_outputs.logits))
+        self.assertTrue(
+            torch.allclose(onnx_outputs.logits, io_outputs.logits),
```

Comment:
So we need `torch.allclose` because we cannot load properly the right version of TRT (or another Nvidia library)?

Reply:
We use torch.allclose because torch.equal is too strict to compare floats.